scripts/build/tests.mjs

```diff
@@ -128,21 +131,28 @@ export async function runConsoleTests(runJs, defaultReporter, runInParallel, opt
     /** @type {Error | undefined} */
     let error;
 
+    const savedNodeEnv = process.env.NODE_ENV;
+    const savedNodeV8Coverage = process.env.NODE_V8_COVERAGE;
     try {
-        setNodeEnvToDevelopment();
+        process.env.NODE_ENV = "development";
+        if (coverage) {
+            process.env.NODE_V8_COVERAGE = path.resolve(coverageDir, "tmp");

```

Comment:
Is there a benefit from doing this rather than letting `c8` process it? I didn't know that `c8 report` existed before now (I had always run it as a node wrapper).

Reply:
It makes the console output a bit more readable and gives you feedback on test results faster since the `c8 report` conversion happens in a separate step.